### ISSUE TITLE
Cannot attach debugger to sandbox

### ISSUE THREAD

**Issue by stanislavyv** [SqrTT/prophet#358]
**Describe the bug**
Cannot attach debugger to sandbox, it was working fine and today it suddenly stopped. Other features such as upload to sandbox/ watch files are working normally.

**To Reproduce**
1. Go to Run and Debug
2. Click "Attach to Sandbox"
3. See errors in Debug console:
`Unauthorized`
 `undefined`
`Disconecting...`
`Unauthorized`

**Expected behavior**
Debugger should attach to sandbox successfuly

**Screenshots**
![image](https://github.com/SqrTT/prophet/assets/63970354/28c9dc78-586a-45a9-817a-a0543b166441)
![image](https://github.com/SqrTT/prophet/assets/63970354/34d7f019-2f7d-4a9a-b379-8add80deb3b7)

**Desktop (please complete the following information):**
 - OS: [windows 11]
 - VSCode version [v1.78.2]
 - The extension version [v1.4.40]

**Additional context**
I can log into my sandbox with the credentials in dw.json
I've tried disabling/ enabling the extension, restarting VSCode, restarting my laptop and changing values in dw.json (to log in another sandbox)

Stack trace:
```
From client: initialize({"clientID":"vscode","clientName":"Visual Studio Code","adapterID":"prophet","pathFormat":"path","linesStartAt1":true,"columnsStartAt1":true,"supportsVariableType":true,"supportsVariablePaging":true,"supportsRunInTerminalRequest":true,"locale":"en","supportsProgressReporting":true,"supportsInvalidatedEvent":true,"supportsMemoryReferences":true,"supportsArgsCanBeInterpretedByShell":true,"supportsMemoryEvent":true,"supportsStartDebuggingRequest":true})
To client: {"event":"prophet.getdebugger.config","type":"event","seq":1}
From client: DebuggerConfig({"config":{"hostname":"***********","username":"***********","password":"***********","version":"test","cartridgesPath":"app_storefront_base:modules","configFilename":{"$mid":1,"external":"file:///c%3A/Users/stani/Dev/ME_REF_APP/me-refapp/dw.json","path":"**********"
To client: {"seq":0,"type":"response","request_seq":1,"command":"initialize","success":true,"body":{"supportsConditionalBreakpoints":true,"supportsHitConditionalBreakpoints":false,"supportsFunctionBreakpoints":false,"supportsConfigurationDoneRequest":true,"supportsEvaluateForHovers":false,"supportsStepBack":false,"supportsSetVariable":false,"supportsRestartFrame":false,"supportsStepInTargetsRequest":false,"supportsGotoTargetsRequest":false,"supportsCompletionsRequest":true,"supportsRestartRequest":false,"supportsExceptionOptions":false,"supportsValueFormattingOptions":true,"supportsExceptionInfoRequest":false,"supportTerminateDebuggee":false,"supportsDelayedStackTraceLoading":false,"supportsLoadedSourcesRequest":false,"supportsLogPoints":false,"supportsTerminateThreadsRequest":false,"supportsSetExpression":false,"supportsTerminateRequest":false,"exceptionBreakpointFilters":[]}}
To client: {"seq":0,"type":"response","request_seq":2,"command":"DebuggerConfig","success":true}
From client: launch({"type":"prophet","request":"launch","name":"Attach to Sandbox","trace":true,"__configurationTarget":6,"__sessionId":"4d8b3372-2930-4eb9-b4f9-761f7a251ca3"})
log -> retriesLeft: 3
log -> retriesLeft: 2
To client: {"seq":0,"type":"event","event":"terminated"}
To client: {"seq":0,"type":"event","event":"output","body":{"category":"console","output":"Unauthorized\n undefined"}}
Unauthorized
 undefined
From client: disconnect({"restart":false})
To client: {"seq":0,"type":"event","event":"output","body":{"category":"console","output":"Disconnecting...\n"}}
Disconnecting...
log -> retriesLeft: 3
log -> retriesLeft: 2
To client: {"seq":0,"type":"event","event":"output","body":{"category":"console","output":"Unauthorized\n"}}
Unauthorized
To client: {"seq":0,"type":"response","request_seq":4,"command":"disconnect","success":true}
```

**Comment by SqrTT** [comment]
please check existing issues, for instance this one #357

TLDR: email is case sensitive for debugger while is not sensitive for uploader

**Comment by stanislavyv** [comment]
It worked! Thank you!
